packages/file-storage/test/file-storage-fs.e2e-spec.ts

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable max-nested-callbacks */
+/* eslint-disable max-lines-per-function */
+import { Test, TestingModule } from '@nestjs/testing';
+import * as dotenv from 'dotenv';
+import { mkdir, rm } from 'fs/promises';
+import { Readable } from 'node:stream';
+import { pipeline } from 'node:stream/promises';
+import { resolve } from 'path';
+
```

Comment:
since you imported some native node packages with the `node:` protocol you might as well go all in and do it for `path` and `fs/promises` as well.